CMakeLists.txt

```diff
@@ -0,0 +1,122 @@
+
```

Comment:
Tiny nit-pick/question: is there a reason you're leaving an empty line at the top of the CMake files? Same goes for the `cmake/modules/SwiftSupport.cmake` file below.

Reply:
Yeah, ideally, someone would do the hard work of figuring out how to add the authorship information to the CMake content.  I've been lazy and this is something that should be done at some point - it is not for any functional reason, only so that there is uniformity in all the files across the repositories.